aircraft/breguet_range.py

```diff
@@ -0,0 +1,53 @@
+"""Breguet range model"""
+from gpkit.shortcuts import Var, Model
+from gpkit.tools import te_exp_minus1
+
+class BreguetRange(Model):
+
+    """Breguet Range Model
+
+    Assumptions
+    -----------
+    Fuel burn varies linearily with thrust and is independent of velocity.
```

Comment:
@mjburton11, here's an example of what @bqpd is talking about with specifying units: https://github.com/aeroa/gpkit-models/blob/master/troposphere.py

... but the difference I see is that the defaults in the Troposphere model are really unlikely to be changed -- they're sort of accepted physical constants. Here, the defaults have a lot more uncertainty associated with them, and will almost certainly need to be fiddled with by users or other models. So I'm wondering if they should perhaps not be defaults.

We need some best practices here.